core/CMakeLists.txt

```diff
@@ -237,6 +237,11 @@ if (runtime_cxxmodules)
   list(APPEND core_implicit_modules "-mByproduct" "ROOT_Rtypes")
 endif(runtime_cxxmodules)
 
+# These are already part of other modules
+list(REMOVE_ITEM Clib_dict_headers strlcpy.h snprintf.h)
+list(REMOVE_ITEM Foundation_dict_headers ESTLType.h TClassEdit.h ThreadLocalStorage.h ROOT/RStringView.hxx)
+list(REMOVE_ITEM Meta_dict_headers TIsAProxy.h TVirtualIsAProxy.h)
```

Comment:
Do we get them because of globbing?

Reply:
No, see `CMakeList.txt`? They are part of variables.

Reply:
That's probably not what we want because we essentially will exclude these header file from building a dictionary. Here likely that's not an issue because they are transiently included.

Instead I suggest to add a check [here](https://github.com/root-project/root/blob/aeb5eee9ec84c061aa55f5b6152a2141152e2520/core/dictgen/src/rootcling_impl.cxx#L3924) which checks if `TLM` is part of `gOptModuleByproducts`. This means that if the suggested module was part of the modules that we build as a module byproducts we should not report that as an issue...

Reply:
Good point. https://github.com/root-project/root/pull/12067 should do what you suggest!